.github/workflows/ci-ubuntu-p4tc-stf.yml

```diff
@@ -11,10 +11,10 @@ on:
 
 jobs:
   # Run p4tc stf tests
-  test-ubuntu18-p4tc-stf:
-    # Only run on pull requests with the "p4tc" label.
-    if: ${{ github.event_name == 'schedule' || contains(github.event.pull_request.labels.*.name, 'p4tc') }}
-    runs-on: ubuntu-20.04
+  test-ubuntu-p4tc-stf:
+    # Only run on pull requests with the "run-p4tc" label.
+    if: ${{ github.event_name == 'schedule' || contains(github.event.pull_request.labels.*.name, 'run-p4tc') }}
```

Comment:
I am not sure an extra label makes sense here. We already have `p4tc` and the test should be triggered by this label.

I am also planning to add a trigger that the tests are triggered by any change in the `p4tc` folder.

Reply:
commit 4 changes label triggering runs back to 'p4tc'